docs/source/contents/4_draft_mapping.rst

```diff
@@ -690,15 +690,91 @@ typeOfResource
 classification
 ==============
 
-relateItem
-==========
+Use case
+^^^^^^^^
```

Comment:
This is the wrong title level based on the context throughout the rest of the draft mapping doc.

This needs to be the following, or things won't build:

```
Use case
--------

```